### PR TITLE
chore(slider): updated handle ui to s1 design

### PR DESCRIPTION
**Pull request by Rajdeepc**
## Description
Update `sp-slider` handle to S1 design

Screenshot 2024-02-14 at 3 48 28 PM
Screenshot 2024-02-14 at 3 48 34 PM

## Related issue(s)

- With the design team pushing back on the recent Slider Handle changes, and it being a blocker for PsWeb catching up to more recent released of SWC.
CSS: Current implementation is correct
SWC: May eventually be this way in S2 but we need to update for S1

## How has this been tested?

-   [x] _Test case 1_
    1. Go [here](https://slider-handleui--spectrum-web-components.netlify.app/storybook/?path=/story/slider--default)

## Types of changes

-   [x] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to change)
-   [ ] Chore (minor updates related to the tooling or maintenance of the repository, does not impact compiled assets)

## Checklist

-   [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
-   [x] My code follows the code style of this project.
-   [ ] If my change required a change to the documentation, I have updated the documentation in this pull request.
-   [x] I have read the **[CONTRIBUTING](<(https://github.com/adobe/spectrum-web-components/blob/main/CONTRIBUTING.md)>)** document.
-   [ ] I have added tests to cover my changes.
-   [x] All new and existing tests passed.
-   [x] I have reviewed at the Accessibility Practices for this feature, see: [Aria Practices](https://www.w3.org/TR/wai-aria-practices/)

## Best practices

This repository uses conventional commit syntax for each commit message; note that the GitHub UI does not use this by default so be cautious when accepting suggested changes. Avoid the "Update branch" button on the pull request and opt instead for rebasing your branch against `main`.

### CODE REVIEW

**Comment by Westbrook** [comment]
Sometimes when there are a lot of changes it take a couple of times to shake out the VRT update. Please take another look at that so we can get this passing across the board.

**Comment by Rajdeepc** [comment]
> Sometimes when there are a lot of changes it take a couple of times to shake out the VRT update. Please take another look at that so we can get this passing across the board.

@Westbrook Good to land now.

File: packages/slider/src/Slider.ts [hunk 1]
```diff
@@ -410,6 +410,15 @@ export class Slider extends SizedMixin(ObserveSlotText(SliderHandle, ''), {
         `;
     }
 
+    private renderHandle(): TemplateResult {
+        if (this.variant === 'tick') {
+            return html``;
+        }
+        return html`
+            ${this.handleController.render()}
+        `;
```
Comment (spdev3000) [inline packages/slider/src/Slider.ts:419]: NIT: Can be shortened to: 
```
return this.variant === 'tick' ?  html`` : this.handleController.render()
```

**Review by spdev3000 — APPROVED** [review]
lgtm

File: packages/slider/src/Slider.ts (hunk 1 above)
Comment (Westbrook) [reply packages/slider/src/Slider.ts:419]: I'd prefer this as well.

**Review by Westbrook — COMMENTED** [review]
Please update one of the stories to have a minimum or maximum value by default to make it easier to track this specific style of change in the future.

**Review by spdev3000 — APPROVED** [review]
lgtm, thx!

File: packages/slider/src/Slider.ts [hunk 2]
```diff
@@ -410,6 +410,12 @@ export class Slider extends SizedMixin(ObserveSlotText(SliderHandle, ''), {
         `;
     }
 
+    private renderHandle(): TemplateResult {
+        return html`
+            ${this.variant === 'tick' ? html`` : this.handleController.render()}
+        `;
```
Comment (Westbrook) [inline packages/slider/src/Slider.ts:416]: At some point we should do focused testing on whether the use of stacked templates here causes us any performance costs, but the small scale use here doesn't seem to be tweaking the performance of the component itself, so :shipit:

**Review by Westbrook — APPROVED** [review]

**Pull request merged**